src/soundbase.cpp

```diff
@@ -166,24 +166,35 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ).arg ( strSystemDriverTechniqueName ) +
-                                    "</b><br><br>" + tr ( "These are all the available drivers with error messages:" ) + "<ul>";
+            QString sErrorMessage = tr ( "<b>Could not find compatible sound driver (using %1).</b><br><br>" ).arg ( strSystemDriverTechniqueName );
```

Comment:
The "... (using ASIO)." seems a bit confusing here as it implies a driver might be found using something else. Is there a reason we're saying this?

Reply:
It won't necessarily say "ASIO" - it provides meaningful information for diagnosing a problem preventing the use of the program.

Reply:
OK, but we have a button that says "Open ASIO settings", so is that button conditional on what `%1` is? This PR is to help people who aren't technical deal with the problem. If we qualify an error message in this way, they'd be excused for wondering why it's qualified, should they be using something else, etc.

Reply:
No -- it only appear on Windows.  (It should also only appear for ASIO -- that needs fixing.)

Reply:
I still think "Could not find compatible sound driver (using ASIO)" feels a bit odd. Like it's expecting them to switch to something that isn't ASIO where the problem might not arise.

Reply:
But that's entirely true.  Switching to Jamulus for Jack on Windows might solve the problem.

Reply:
It would be less mysterious to just make it the subject in that case and say, "Could not find a compatible ASIO sound driver." That is, unless we want also to say "Maybe you should try using [other system]." which I doubt.

Reply:
Maybe go back to `No usable %1 audio device found.` then?

Reply:
That reads better than a parenthesised suffix (I think). 

BTW re. "audio device", it's probably the subject of a different ticket, but for such an important thing we should keep to a standard term. The [style guide](https://jamulus.io/contribute/Style-and-Tone) says we should be using "sound hardware" here but "audio device" does feel a bit more conventional somehow.

Reply:
Ok. Fixed that.

> The whole flow is really bad, I think - huge block of platform-specific code at level that should be platform-independent.

I'm a bit unsure, but I remember there was a discussion about fixing this. Wasn't @npostavs involved?

Reply:
I guess pljones' remark at https://github.com/jamulussoftware/jamulus/pull/1378#issuecomment-812866670 is similar.